com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/liquibase/PopulateGUIDColumnConfigurationRegistryChange.java

```diff
@@ -0,0 +1,125 @@
+package com.sap.cloud.lm.sl.cf.core.liquibase;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.cloudfoundry.client.lib.CloudCredentials;
+import org.cloudfoundry.client.lib.domain.CloudSpace;
+import org.cloudfoundry.client.lib.rest.CloudControllerClient;
+import org.cloudfoundry.client.lib.rest.CloudControllerClientFactory;
+
+import com.sap.cloud.lm.sl.cf.core.message.Messages;
+import com.sap.cloud.lm.sl.cf.core.model.CloudTarget;
+import com.sap.cloud.lm.sl.cf.core.util.Configuration;
+import com.sap.cloud.lm.sl.cf.core.util.SecurityUtil;
+
+public class PopulateGUIDColumnConfigurationRegistryChange
+    extends AbstractDataTransformationChange<Map<Long, CloudTarget>, Map<Long, String>> {
+
+    private static final String SELECT_STATEMENT = "SELECT ID, TARGET_SPACE, TARGET_ORG FROM CONFIGURATION_REGISTRY LIMIT 100";

```

Comment:
Wouldn't 'LIMIT 100' make sure this change only works for the first 100 configuration entries? What about the rest? Maybe you should also use 'OFFSET X' if you want to implement some type of paging here.